app/assets/javascripts/analytics/_pageViews.js

```diff
@@ -1,7 +1,23 @@
 (function (GOVUK) {
   GOVUK.GDM.analytics.pageViews = {
     'init': function () {
+      this.setCustomDimensions()
       GOVUK.analytics.trackPageview();
+    },
+
+    'setCustomDimensions': function() {
+      var eligibility;
+      
+      if (/^\/suppliers\/opportunities\/(\d+)\/responses\/create$/.test(GOVUK.GDM.analytics.location.pathname())) {
```

Comment:
Hey @tombye, how do we feel about doing hardcoded URL strings in our Javascript?  
This was [something that @ris brought up before](https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/380#discussion_r74052889) when you were away, and I don't really know how we feel about it.